charts/schema-registry/templates/configmap.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "schema-registry.configmap" . }}
+data:
+  "CONTROLLER_URL": "tcp://{{ .Values.controller.ip }}:{{ .Values.controller.port }}"
+  "AUTHORIZATION_ENABLED": "{{ .Values.authentication.enabled }}"
```

Comment:
`AUTH_ENABLED` would be more appropriate, as it represents both authentication and authorization.

Reply:
This name is based on the name that is used in the code
https://github.com/pravega/schema-registry/blob/for-test/server/src/conf/schema-registry.config.properties#L32